Build failure on arm FreeBSD

The build fails as follows on arm FreeBSD:

```
# github.com/go-piv/piv-go/piv
vendor/github.com/go-piv/piv-go/piv/pcsc_freebsd.go:29:15: 0x8010002E (untyped int constant 2148532270) overflows int32
golang.org/x/crypto/ssh/agent
*** Error code 2
```

Please check and possible add a cast to make the number negative.

--- a/piv/pcsc_freebsd.go
+++ b/piv/pcsc_freebsd.go
@@ -26,5 +26,5 @@ func scCheck(rc C.long) error {
 }
 
 func isRCNoReaders(rc C.long) bool {
-	return rc == 0x8010002E
+	return uint32(rc) == 0x8010002E
 }

